.github/workflows/renovate-go-mod-fix.yaml

```diff
@@ -0,0 +1,22 @@
+name: go-mod-fix
+on:
+  push:
+    branches:
+      - renovate/*
+
+jobs:
+  go-mod-fix:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: fix
+        uses: at-wat/go-sum-fix-action@v0
+        with:
+          git_user: Pion Bot
+          git_email: 59523206+pionbot@users.noreply.github.com
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
As CI is running on GitHub Actions on this repo, push by this token doesn't trigger CI due to the GitHub Actions' limitation. Needed to register and use another token.

Reply:
fixed